Source/santad/DataLayer/SNTRuleTable.m

```diff
@@ -117,6 +117,8 @@ + (NSArray *)criticalSystemBinaryPaths {
       @"/Applications/Santa.app/Contents/MacOS/santasyncservice",
     ]];
 
+    // TODO for macOS 13 add the cryptexes prefix
```

Comment:
Add a link to a GH issue for this

Reply:
I've created #886 but this was a left-over comment from other refactoring,  so I've removed it.